### PR TITLE
Update ltr 3.4.14

### PR DESCRIPTION
**Pull request by ocefpaf**
Alternative for #102

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-gdal:
-- 2.4
-libgdal:
-- 2.4
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: @SrNetoChan do you know if the LTR builds against `gdal 3`? If not we would have to rebuild `gdal 2.4.x` against newer dependencies.

Comment (SrNetoChan) [reply recipe/conda_build_config.yaml]: 3.4 should be kept at gdal 2.4.x It's not ready for gdal 3

**Comment by ocefpaf** [comment]
> 3.4 should be kept at gdal 2.4.x It's not ready for gdal 3

OK. I'm removing the binaries and I'll work on a compatible gdal 2.4. Thanks!

**Comment by SrNetoChan** [comment]
@ocefpaf FYI the LTR branch will be updated to 3.10 in February 21, so if it's too much trouble, you can just delete the binaries and don't apply any patches to 3.4 until then.

**Comment by ocefpaf** [comment]
> @ocefpaf FYI the LTR branch will be updated to 3.10 in February 21, so if it's too much trouble, you can just delete the binaries and don't apply any patches to 3.4 until then.

Good to know. Thanks! I'll try with the new [gdal 2.4.4](https://github.com/conda-forge/gdal-feedstock/pull/364) though and see how far we get.

**Pull request merged**